+ source /root/mugen/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start environmental preparation.'
+ message='Start environmental preparation.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start environmental preparation.'
Thu Jul 27 17:04:31 2023 - INFO  - Start environmental preparation.
+ SSH_CMD 'useradd sshuser -d /home/sshuser -p ' '' '' ''
+ cmd='useradd sshuser -d /home/sshuser -p '
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c 'useradd sshuser -d /home/sshuser -p ' -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 17:04:33 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ LOG_INFO 'End of environmental preparation!'
+ message='End of environmental preparation!'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End of environmental preparation!'
Thu Jul 27 17:04:34 2023 - INFO  - End of environmental preparation!
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Thu Jul 27 17:04:35 2023 - INFO  - Start to run test.
+ expect
spawn ssh @
usage: ssh [-46AaCfGgKkMNnqsTtVvXxYy] [-B bind_interface]
           [-b bind_address] [-c cipher_spec] [-D [bind_address:]port]
           [-E log_file] [-e escape_char] [-F configfile] [-I pkcs11]
           [-i identity_file] [-J [user@]host[:port]] [-L address]
           [-l login_name] [-m mac_spec] [-O ctl_cmd] [-o option] [-p port]
           [-Q query_option] [-R address] [-S ctl_path] [-W host:port]
           [-w local_tun[:remote_tun]] destination [command [argument ...]]
expect: spawn id exp3 not open
    while executing
"expect {
            "password" {
                send "\r"
            }
        }"
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_openssh_ssh_user.sh line 42'
+ message='oe_test_openssh_ssh_user.sh line 42'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_openssh_ssh_user.sh line 42'
Thu Jul 27 17:04:37 2023 - ERROR - oe_test_openssh_ssh_user.sh line 42
+ '[' 0 -eq 1 ']'
+ return 0
+ expect
spawn ssh sshuser@
ssh: Could not resolve hostname : Name or service not known
expect: spawn id exp3 not open
    while executing
"expect {
            "password" {
                send "\r"
            }
        }"
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_openssh_ssh_user.sh line 57'
+ message='oe_test_openssh_ssh_user.sh line 57'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_openssh_ssh_user.sh line 57'
Thu Jul 27 17:04:39 2023 - ERROR - oe_test_openssh_ssh_user.sh line 57
+ '[' 0 -eq 1 ']'
+ return 0
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Thu Jul 27 17:04:40 2023 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 2
+ test 2 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Thu Jul 27 17:04:41 2023 - ERROR - failed to execute the case.
+ exit 2
+ post_test
+ LOG_INFO 'Start to restore the test environment.'
+ message='Start to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to restore the test environment.'
Thu Jul 27 17:04:42 2023 - INFO  - Start to restore the test environment.
+ SSH_CMD 'userdel -r sshuser' '' '' ''
+ cmd='userdel -r sshuser'
+ remoteip=
+ remotepasswd=
+ remoteuser=
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c 'userdel -r sshuser' -i '' -u '' -p '' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 17:04:44 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ LOG_INFO 'End to restore the test environment.'
+ message='End to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to restore the test environment.'
Thu Jul 27 17:04:45 2023 - INFO  - End to restore the test environment.
